context/app/static/js/components/workspaces/JobLink.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useContext } from 'react';
+
+import { AppContext } from 'js/components/Providers';
+import OutboundIconLink from 'js/shared-styles/Links/iconLinks/OutboundIconLink';
+import { startJob } from './utils';
+
+function JobLink({ workspace, job, children }) {
+  const { workspacesEndpoint, workspacesToken } = useContext(AppContext);
+
+  function createHandleStart(workspaceId) {
+    async function handleStart() {
+      startJob({ workspaceId, workspacesEndpoint, workspacesToken });
+      // TODO: Open new tab
+      // eslint-disable-next-line no-alert
+      alert('TODO: Open a new tab that will poll until the job is started.');
+    }
+    return handleStart;
+  }
+
+  if (job.allowNew) {
+    const handleStart = createHandleStart(workspace.id);
+    return (
+      <OutboundIconLink>
+        <button onClick={handleStart} type="submit" style={{ all: 'unset' }}>
+          {children}
+        </button>
+      </OutboundIconLink>
```

Comment:
```suggestion
    return (
      <OutboundIconLink  component="button" onClick={() => handleStart(workspace.id)} type="submit" style={{ all: 'unset' }}>
          {children}
      </OutboundIconLink>
```

Reply:
I think I've applied all the other suggestions above, so I have
```
      <OutboundIconLink>
        <LinkButton onClick={createHandleStart(workspace.id)}>{children}</LinkButton>
      </OutboundIconLink>
```
but `component={LinkButton}` doesn't produce the same result.

Reply:
It should be `component={button}`.

Reply:
There should only be one component which is `OutboundIconLink`.

Reply:
If that doesn't work, feel free to merge with whatever you think is best and I can pick it up later.